react-components/src/components/View/Image360HistoricalDetails.tsx

```diff
@@ -34,14 +34,14 @@ export const Image360HistoricalDetails = ({
   >([]);
   const [imageUrls, setImageUrls] = useState<(string | undefined)[]>([]);
   const [minWidth, setMinWidth] = useState('100px');
+  const [scrollPosition, setScrollPosition] = useState(0);
 
   useEffect(() => {
-
     const fetchRevisionCollection = async () => {
       if (image360Entity) {
         const revisions = image360Entity.getRevisions();
         const revisionDates = revisions.map((revision) => revision.date);
-        const imageDatas = await Promise.all(revisions.map( async revision => await revision.getPreviewThumbnailUrl()));
+        const imageDatas = await Promise.all(revisions.map(async (revision) => await revision.getPreviewThumbnailUrl()));
```

Comment:
This makes me think we should have a prettier-file for this component too :P But we can have that in another PR